src/views/Delegate.vue

```diff
@@ -9,7 +9,7 @@
         <h1 v-if="loaded" v-text="$t('delegate.header')" />
       </div>
       <template v-if="loaded">
-        <Block :itle="$t('delegate.selectAddress')">
+        <Block :title="$t('delegate.selectAddress')">
```

Comment:
"Select address" is relevant? there is no dropdown right 🤔 ..  i think maybe it should be "Enter address" or "Enter an address to delegate"
![image](https://user-images.githubusercontent.com/15967809/124561094-20e1ef00-de5b-11eb-8ee6-5de01b6a7d26.png)

Reply:
Ok I'll make a card

Reply:
Please approve if there's nothing else